code/game/objects/effects/decals/turfdecal/markings.dm

```diff
@@ -343,9 +343,9 @@
 /obj/effect/turf_decal/siding/wood/end
 	icon_state = "siding_wood_end"
 
-/obj/effect/turf_decal/siding/thinplating
+/obj/effect/turf_decal/siding/thinplating // Variants formerly known as /siding/thinplating_new/
```

Comment:
```suggestion
/obj/effect/turf_decal/siding/thinplating
```

this comment is useless now and will become even more useless in 3 months from now, so we try to avoid including legacy comments in our code. just delete it and we're fine

Reply:
you got it chief